package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-create-ui",
-  "version": "2.0.17",
+  "version": "2.0.18",
```

Comment:
Another PR was just merged with this version number. You may have to update this. (I'm not sure why it's not showing as a conflict!?)

Reply:
ok. Updated to 2.0.19 
anyway!